core/src/main/java/org/elasticsearch/action/fieldstats/FieldStats.java

```diff
@@ -550,27 +625,50 @@ public static FieldStats readFrom(StreamInput in) throws IOException {
         long sumTotalTermFreq = in.readLong();
         boolean isSearchable = in.readBoolean();
         boolean isAggregatable = in.readBoolean();
-
+        boolean hasMinMax = true;
```

Comment:
Yeah, another PR is totally fine. No need to hold this up. And I'm not 100% sure it makes sense to convert them, just that it makes sense to think about it.

We have lots of NamedWriteables that write things in common - queries and aggregations do it all the time. They defer to superclasses to load the common stuff.